packages/neo-one-node-http-rpc/src/rpcServer$.ts

```diff
@@ -70,6 +87,14 @@ export const rpcServer$ = ({
       const rpcMiddleware = rpc({ blockchain, node });
       router.use(context({ monitor }));
 
+      if (!(rateLimitOptions !== undefined && rateLimitOptions.disabled)) {
```

Comment:
This is really hard to read, generally you want to avoid negating a compound boolean expression. Instead, make this `rateLimitOptions === undefined || !rateLimitOptions.disabled`

Reply:
I'll fix them, but my buddy Augustus De Morgan would like I word.